### DISCUSSION TITLE
What is the reason of using `activeEditor` instead of `editor` from `useLexicalComposerContext`?

### DISCUSSION THREAD

**Discussion by hobroker** [facebook/lexical#5241] [General]
I am building my own editor using some stuff from the playground and I see usages of `activeEditor`, which is set like this
```js
editor.registerCommand(
  SELECTION_CHANGE_COMMAND,
  (_, newEditor) => {
    setActiveEditor(newEditor);
    return false;
  },
  COMMAND_PRIORITY_CRITICAL,
)
```

It's been used for commands like `activeEditor.dispatchCommand(UNDO_COMMAND, undefined);`

I tried using `editor` from `const [editor] = useLexicalComposerContext();` and it works the same, so I'm wondering why would I use `activeEditor`? Would I stumble upon some issues in the future if I keep using the `editor` from the context?

**Comment by ivailop7** [comment]
You  can have multiple editors on the same page, the use case for this on the playground is the comment plugin, where the comment entry is a different editor than the main document one, so you want to toggle the active editor with its plugins, etc.

**Reply by hobroker** [reply]
That makes sense. Thank you!
